json2jargon.sh

```diff
@@ -0,0 +1,25 @@
+cat data.json | jq -r '
+def fix_links: if . then gsub(":[^{}]+\\}"; "}") | gsub("\n"; "\n   ") else . end;
```

Comment:
The English and Finnish scripts are different as the Finnish script defaults to English when a field is empty, while the English script doesn't, but I don't think it would be harmful if the English script "defaulted to English" as well. I'll see if I can refactor them into one script.